backend/clubs/management/commands/goap_import.py

```diff
@@ -43,7 +53,11 @@ def handle(self, *args, **kwargs):
         if self.skip_tags:
             self.stdout.write("Skipping tag imports...")
         self.process_url(self.START_URL)
-        self.stdout.write("Imported {} clubs!".format(self.club_count))
+        self.stdout.write(
+            "Imported {} clubs! {} created, {} updated".format(
```

Comment:
Now that we're on 3.6 and up, should we start using format strings for string interpolation?

Reply:
I like that, I'll switch it over!